src/components/FlowRunTimeline.vue

```diff
@@ -1,6 +1,34 @@
 <template>
-  <div class="flow-run-timeline">
+  <div
+    class="flow-run-timeline"
+    :class="{ 'flow-run-timeline--fullscreen': isFullscreen }"
+    tabindex="0"
+    aria-label="Flow run timeline graph"
+    @keydown.c="recenterGraph"
+    @keydown.f="toggleFullscreen"
+    @keydown.a="updateHideEdges(!hideEdges)"
+  >
+    <p-button
+      v-if="isFullscreen"
+      class="flow-run-timeline__fullscreen-exit"
+      icon="XIcon"
+      flat
+      size="lg"
+      @click="toggleFullscreen"
+    />
     <div class="flow-run-timeline__options">
```

Comment:
These buttons could use a tiny bit of padding between them, I think the outlines of the button children are overlapping each other atm